src/importexport/mei/internal/meiimporter.cpp

```diff
@@ -2207,6 +2218,80 @@ bool MeiImporter::readDynam(pugi::xml_node dynamNode, Measure* measure)
     return true;
 }
 
+/**
+ * Read a f (FiguredBassItem)
+ */
+
+bool MeiImporter::readF(pugi::xml_node fNode, engraving::FiguredBass* figuredBass)
+{
+    IF_ASSERT_FAILED(figuredBass) {
+        return false;
+    }
+
+    bool warning;
+    libmei::F meiF;
+    meiF.Read(fNode);
+
+    const int line = static_cast<int>(figuredBass->itemsCount());
+    FiguredBassItem* figuredBassItem = figuredBass->createItem(line);
+    if (!figuredBassItem) {
+        return true;
+    }
```

Comment:
It looks like `createItem` never returns nullptr, so no need to check for it. (Also, I wondered why there is 'return true" here, while this would actually be a failure situation, wouldn't it?)

Reply:
Returning `false` means that the import fails and cannot go through - the files is critically corrupted. Returning `true` is meant to allow for files to be imported even if warnings are yielded (e.g. in force mode).